index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<title>Ecxercise 3</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="nav-bar"> <!-- Flex parent -->
```

Comment:
these comments are quite nice but we don't tend to comment HTML code in practice very much, only when explaining something that isn't easily understandable without a comment, so i would recommend removing the comments, again they are nicely done but just getting closer to the way everyone does it in industry